packages/playground/blueprints/src/lib/steps/mkdir.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { NodePHP } from '@php-wasm/node';
+import { RecommendedPHPVersion } from '@wp-playground/wordpress';
+import { mkdir } from './mkdir';
+
+const docroot = '/php';
+describe('Blueprint step mkdir', () => {
+	let php: NodePHP;
+	beforeEach(async () => {
+		php = await NodePHP.load(RecommendedPHPVersion);
+	});
+
+	it('should create a directory', async () => {
+		expect(php.fileExists(`/${docroot}`)).toBe(false);
```

Comment:
`/${docroot}` will result in a double slash. Also, the word `docroot` has a specific meaning in Playground – it's the document root for serving files. In here, however, this just a directory. Let's call it something else, maybe `directoryPath` or so.

Reply:
Refactored for maximum clarity.